app/models/waste_carriers_engine/order_item_log.rb

```diff
@@ -6,6 +6,7 @@
 module WasteCarriersEngine
   class OrderItemLog
     include Mongoid::Document
+    include CanCheckRegistrationStatus
```

Comment:
Can this be dropped now that `active_registration?` has been added?

Reply:
Yes. Removing...